polkadot/node/core/pvf/src/execute/worker_interface.rs

```diff
@@ -87,6 +87,9 @@ pub enum Outcome {
 	/// a trap. Errors related to the preparation process are not expected to be encountered by the
 	/// execution workers.
 	InvalidCandidate { err: String, idle_worker: IdleWorker },
+	/// The error is probably transient. It may be for example because the wasm runtime version
+	/// in the worker is different from the wasmtime version of the host.
```

Comment:
That's not a correct description, the host doesn't (hopefully) operate Wasmtime itself, it's workers' business. The correct one could be like "because the artifact was prepared with a Wasmtime version different from the version in the current execution environment" or something like that. Or just a link to the issue.

Reply:
Exactly, correct! Thank you!) fixed https://github.com/paritytech/polkadot-sdk/pull/3187/commits/3f3e132db9c1ff7e44a6e5a431e6fc4091f4faf0